src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1567,5 +1541,33 @@ internal enum ErrorCode
         // Available  = 8355-8399
 
         ERR_FeatureNotAvailableInVersion8 = 8400,
+
+        WRN_NullAsNonNullable = 8600,
+        WRN_NullReferenceAssignment = 8601,
+        WRN_NullReferenceReceiver = 8602,
+        WRN_NullReferenceReturn = 8603,
+        WRN_NullReferenceArgument = 8604,
+        HDN_NullCheckIsProbablyAlwaysTrue = 8605,
+        HDN_NullCheckIsProbablyAlwaysFalse = 8606,
+        HDN_ExpressionIsProbablyNeverNull = 8607,
+        WRN_NullabilityMismatchInTypeOnOverride = 8608,
+        WRN_NullabilityMismatchInReturnTypeOnOverride = 8609,
+        WRN_NullabilityMismatchInParameterTypeOnOverride = 8610,
+        WRN_NullabilityMismatchInParameterTypeOnPartial = 8611,
+        WRN_NullabilityMismatchInTypeOnImplicitImplementation = 8612,
+        WRN_NullabilityMismatchInReturnTypeOnImplicitImplementation = 8613,
+        WRN_NullabilityMismatchInParameterTypeOnImplicitImplementation = 8614,
+        WRN_NullabilityMismatchInTypeOnExplicitImplementation = 8615,
+        WRN_NullabilityMismatchInReturnTypeOnExplicitImplementation = 8616,
+        WRN_NullabilityMismatchInParameterTypeOnExplicitImplementation = 8617,
+        WRN_UninitializedNonNullableField = 8618,
+        WRN_NullabilityMismatchInAssignment = 8619,
+        WRN_NullabilityMismatchInArgument = 8620,
+        WRN_NullabilityMismatchInReturnTypeOfTargetDelegate = 8621,
+        WRN_NullabilityMismatchInParameterTypeOfTargetDelegate = 8622,
+        ERR_ExplicitNullableAttribute = 8623,
+        ERR_NotNullableOperatorNotReferenceType = 8624,
+        WRN_NoBestNullabilityArrayElements = 8625,
```

Comment:
>WRN_NoBestNullabilityArrayElements [](start = 8, length = 34)

I don't see where this error is reported in this commit.

Reply:
That was an oversight. We should probably report this warning for an array initializer rather than one warning per element mismatch. Added `// PROTOTYPE` comment.

---
In reply to: [169862843](https://github.com/dotnet/roslyn/pull/24935#discussion_r169862843) [](ancestors = 169862843)